src/Sushi.Mediakiwi/Presentation/Logic/Navigation.cs

```diff
@@ -573,21 +575,26 @@ internal static string GetTabularTagNewDesign(Beta.GeneratedCms.Console containe
                             }
                             else
                             {
-                                IComponentListTemplate cl = t.List.GetInstance(container.Context);
+                                // tab is one of the other lists in the tablist
+                                IComponentListTemplate cl = tab.List.GetInstance(container.Context);
                                 if (cl != null)
                                 {
                                     cl.wim.Console = master;
 
                                     int group2Id = Utility.ConvertToInt(container.Request.Query["group2"]);
                                     int group2ElementId = Utility.ConvertToInt(container.Request.Query["group2item"]);
 
-                                    if (t.List.ID == group2Id)
+                                    // trigger list load if desired
+                                    if (tab.TiggerListLoad)

```

Comment:
@mark-devries Shouldn't this be an OR statement, like:
`if (tab.List.ID == group2Id || tab.TiggerListLoad)`
this will reduce the need for an additional if else clause

Reply:
this was the original code:
if (tab.List.ID == group2Id)
{
	cl.wim.DoListLoad(group2ElementId, 0);
}
else
{
	cl.wim.DoListLoad(container.Item.Value, 0);
}

which is now this:
if (tab.TiggerListLoad)
{
	if (tab.List.ID == group2Id)
	{
		cl.wim.DoListLoad(group2ElementId, 0);
	}
	else
	{
		cl.wim.DoListLoad(container.Item.Value, 0);
	}
}

ie. in the old situation, ListLoad was always triggered
in the new situation, ListLoad is only triggered if tab.TriggerListLoad == true